templates/components/carousel.html

```diff
@@ -43,4 +49,9 @@
         </div>
     </a>
     {{/each}}
+    {{#if arrows}}
+    {{#if carousel.slides.length '>' 1}}
```

Comment:
Can we concatenate two conditions?

Reply:
I would like to, but do we have a way to do it in Handlebars?